build.zig

```diff
@@ -74,14 +79,6 @@ pub fn build(b: *std.Build) !void {
     try @import("build_examples.zig").build(b, optimize, target, module);
 }
 
-fn sdkPath(comptime suffix: []const u8) []const u8 {
```

Comment:
The removal of `sdkPath` here is not correct.

In some cases where you've removed it, it is correct - but in others it is not because `App` is not used by _this_ build.zig file but rather by the person importing it, and would break usage like that in https://github.com/hexops/mach-core-starter-project